Makefile

```diff
@@ -1,37 +1,80 @@
 # Update this path to your obsidian vault directory
-export OBSIDIAN_PATH = /Users/eugene/obsidian-vault/
-export TRANSFORMER_CACHE = /Users/eugene/.cache/huggingface/hub
+# now you can also plugin the path in yoru make argument or as a env variable
+# echo "export OBSIDIAN_PATH=/path/to/obsidian" >> ~/.bashrc and source ~/.profile 
+export OBSIDIAN_PATH ?= /Users/eugene/obsidian-vault/
+export TRANSFORMER_CACHE ?= /Users/eugene/.cache/huggingface/hub
 
 # These generally do not need to be changed
 PWD_PATH = ${PWD}
 DOCKER_OBSIDIAN_PATH = /obsidian-vault
 NETWORK = obsidian-copilot
 IMAGE_TAG = obsidian-copilot
 
+# Choose your container runtime: docker or podman. Default is docker.
+# echo "export RUNTIME=podman" >> ~/.bashrc and source ~/.profile 
+# if you dont want to keep typing it for each make or change it here
+RUNTIME ?= podman
```

Comment:
Doesn't this set the default runtime to podman? Could we set the default to docker, as listed in line 13, please?